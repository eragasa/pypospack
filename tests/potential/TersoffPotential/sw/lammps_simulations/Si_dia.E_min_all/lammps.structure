# ['Si']

8 atoms
1 atom types

    0.0000     5.4687 xlo xhi
    0.0000     5.4687 ylo yhi
    0.0000     5.4687 zlo zhi

    0.0000     0.0000     0.0000 xy xz yz

Atoms

1 1     0.0000     0.0000     2.7344
2 1     4.1015     4.1015     4.1015
3 1     0.0000     2.7344     0.0000
4 1     4.1015     1.3672     1.3672
5 1     2.7344     0.0000     0.0000
6 1     1.3672     4.1015     1.3672
7 1     2.7344     2.7344     2.7344
8 1     1.3672     1.3672     4.1015
