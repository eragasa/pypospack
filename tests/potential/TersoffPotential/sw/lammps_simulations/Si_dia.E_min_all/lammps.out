version https://git-lfs.github.com/spec/v1
oid sha256:809843ff273202ba1c4da00c0a5beb931350a076a31ef02db45ad55861b14cb9
size 2500
